soc/xtensa/espressif_esp32/esp32s3/default.ld

```diff
@@ -635,6 +634,8 @@ SECTIONS
 #include <zephyr/linker/intlist.ld>
 #endif
 
+_heap_sentry = 0x3fceb910;
```

Comment:
As reference, `_heap_sentry` is set to `PROVIDE ( __stack = .... );` provided in original esp-idf linker files.